tools/ci/inria/main

```
@@ -290,38 +290,45 @@ case $NODE_NAME in
 
-grep -q '^NATIVE_COMPILER=false' Makefile.config && make_native=false
+  grep -q '^NATIVE_COMPILER=false' Makefile.config && make_native=false
 
-if test "$flambda" = "true" && test "$make_native" = "false"; then
-  echo "No need to test flambda in a bytecode-only system; skipping the test."
-  exit 0
-fi
+  if test "$flambda" = "true" && test "$make_native" = "false"; then
+    echo "No need to test flambda in a bytecode-only system; skipping the test."
+    exit 0
+  fi
 
-if $bootstrap; then
-  $make $jobs --warn-undefined-variables core
-  $make $jobs --warn-undefined-variables bootstrap
-  if $make_native; then
-    $make $jobs --warn-undefined-variables opt.opt
+  if $bootstrap; then
+    $make $jobs --warn-undefined-variables core
+    $make $jobs --warn-undefined-variables bootstrap
+    if $make_native; then
+      $make $jobs --warn-undefined-variables opt.opt
+    fi
+  else
+    $make $jobs --warn-undefined-variables
   fi
-else
-  $make $jobs --warn-undefined-variables
-fi
 
+  if $make_native && $check_make_alldepend; then
+    $make --warn-undefined-variables alldepend
+  fi
 
-if $make_native && $check_make_alldepend; then
-  $make --warn-undefined-variables alldepend
-fi
+  $make --warn-undefined-variables install
+  case $confoptions in
+    *--disable-unix-lib*) ;;         # test-in-prefix needs Unix lib
+    *)
+      $make -f Makefile.test -C testsuite/in_prefix test-in-prefix
+      ;;
+  esac
+  rm -rf "$instdir"
+}
 
-$make --warn-undefined-variables install
-case $confoptions in
-  *--disable-unix-lib*) ;;         # test-in-prefix needs Unix lib
-  *)
-    $make -f Makefile.test -C testsuite/in_prefix test-in-prefix
-    ;;
-esac
-rm -rf "$instdir"
+# PATH, but without any homebrew additions (tests vanilla macOS in the build,
+# while still allowing full utilities later in the testsuite)
+CLEAN_PATH=$(echo "$PATH" | tr ':' '\n' | grep -vF '/homebrew/' | paste -sd: -)
```

How about using `$(brew --prefix)/` instead of `/homebrew/`?

---

I was seeing a tiny benefit to just executing this on all the workers, not having to gate it to be macOS specifically (or ignore errors in the command, etc.), given that it's more a smoke-test than anything else.

---

It's also that the `/homebrew` path seems specific to the CI vendor, on my macOS system brew is installed under `/opt/homebrew`.

---

That's not what it's doing, though - it removes every PATH entry which _contains_ `/homebrew/`